examples/angular/angular.json

```diff
@@ -11,8 +11,11 @@
       "prefix": "app",
       "architect": {
         "build": {
-          "builder": "@angular-devkit/build-angular:browser",
+          "builder": "@angular-builders/custom-webpack:browser",
```

Comment:
Can you provide context on this change?

Reply:
yeah, so this allows us to add extra webpack config to the default, which resolves the json config(`amplify_outputs.json`) and the value of `process.env.PATH` at build time.

```js
// examples/angular/extra-webpack.config.js
const webpack = require('webpack');

module.exports = {
  resolve: {
    extensions: ['.json'],
  },
  plugins: [
    new webpack.DefinePlugin({
      'process.env.PATH': JSON.stringify(
        process.env.VERSION === 'gen1' ? 'src/aws-exports' : 'amplify_outputs'
      ),
    }),
  ],
};
```

More info: https://www.npmjs.com/package/@angular-builders/custom-webpack